### ISSUE TITLE
Combining reporting

### ISSUE THREAD

**Issue by mgifford** [w3c/wai-wcag-em-report-tool#100]
Websites are often built with multiple pieces of code. Each one an have an accessibility conformance report (ACR) associated with them.  Could be the framework, the CMS, or 3rd Party widgets used.  There should be a way to integrate the known systems and their ACRs into the accessibility review. 

This is especially important in the first report and after upgrades. This would be a change for WCAG-EM 2.0
